build/bazel/remote/execution/v2/remote_execution.proto

```diff
@@ -1843,7 +1843,10 @@ message CacheCapabilities {
 
 // Capabilities of the remote execution system.
 message ExecutionCapabilities {
-  // Remote execution may only support a single digest function.
+  // Legacy field for indicating which digest function is supported by
+  // the remote execution system. Implementations should consider the
+  // repeated digest_functions field first, falling back to this
+  // singular field if unset.
```

Comment:
Should we be explicit and say "This field MUST be set." or "This field MUST be set to a value other than UNKNOWN." here?

Reply:
I'm not sure we should go that far. Once we're a couple of years down the line and most clients have logic to process both `digest_function` and `digest_functions`, I guess it wouldn't be too bad if a clean slate server implementation doesn't set this field.

Reply:
I think that would be a breaking change which needs a version bump.

In the meantime we already say that digest_function must be set if digest_functions is set, it would be strange to allow neither to be set IMO.

Reply:
Gotcha! Just changed the description of `digest_function` to make it required. PTAL!